stations.csv

```diff
@@ -70497,3 +70497,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74638;Bari Santa Rita;bari-santa-rita;;;41.07330246992781;16.851540354841326;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311250;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74638;
 74639;Lascari;lascari;;;38.011695;13.928291;19906;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S12070;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74639;
 74640;Torino Grosseto;torino-grosseto;;;45.10546665758956;7.666751284086174;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8300097;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74640;
+74641;Antequera Av;antequera-av;7102030;;37.02978;-4.56119;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;02030;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74641;
+74642;A Gudina-Porta Galicia;a-gudina-porta-galicia;7108251;;42.06367;-7.13436;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;08251;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74642;
+74643;Callosa de Segura Cox;callosa-de-segura-cox;7103412;;38.136389;-0.875833;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;03412;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74643;
```

Comment:
Even if Wikipedia suggests a `-` between Callosa de Segura and Cox I think it's better typographically to use an EN-dash.

```suggestion
74643;Callosa de Segura – Cox;callosa-de-segura-cox;7103412;;38.136389;-0.875833;;ES;Europe/Madrid;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;03412;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74643;
```